dev/tests/integration/testsuite/Magento/Customer/_files/customer.php

```diff
@@ -32,3 +32,27 @@
 $customer->isObjectNew(true);
 $customer->save();
 $customerRegistry->remove($customer->getId());
+
```

Comment:
This file is used in many other tests. If you need some specific customer behaviour, please create new fixture.